Dockerfile-proxy

```diff
@@ -1,6 +1,6 @@
-ARG RUNTIME_IMAGE=gcr.io/linkerd-io/base:2017-10-30.01
+ARG RUNTIME_IMAGE=gcr.io/linkerd-io/base:2019-02-08.01
 
-FROM gcr.io/linkerd-io/base:2017-10-30.01 as fetch
+FROM gcr.io/linkerd-io/base:2019-02-08.01 as fetch
```

Comment:
Ah, hmm, it looks like the proxy's watch implementation is having some issues with the new Debian OS. When I run linkerd with TLS enabled, I see a bunch of these warnings in the `linkerd-proxy` logs:

```
WARN admin={bg=tls-config} linkerd2_fs_watch::inotify watch error: Os { code: 22, kind: InvalidInput, message: "Invalid argument" }, polling the fs until next change
```

Note that TLS still works despite the warnings, but IIRC, there's some custom watch code in the proxy that will fallback to polling if inotify is unavailable... or something like that. Am pretty sure that's what the warning is telling us, but you might want to check with the proxy folks to see what's up.

Reply:
This is unexpected. We should figure this out before merging...

Reply:
Filed #2331 to track.

Reply:
I haven't gotten to it yet, but we should also check to make sure that the new base image doesn't break the new CNI setup as well.

Reply:
Confirmed CNI working on this branch...

### Installation

```bash
$ bin/linkerd install-cni | kubectl apply -f -
$ bin/linkerd install --linkerd-cni-enabled | kubectl apply -f -
$ curl https://run.linkerd.io/emojivoto.yml | bin/linkerd inject --linkerd-cni-enabled - |   kubectl apply -f -
```

### Daemonset output

```bash
$ kubectl -n linkerd logs -f pod/linkerd-cni-8qgc7
Wrote linkerd CNI binaries to /host/opt/cni/bin
Using CNI config template from CNI_NETWORK_CONFIG environment variable.
      "k8s_api_root": "https://__KUBERNETES_SERVICE_HOST__:__KUBERNETES_SERVICE_PORT__",
      "k8s_api_root": "https://10.96.0.1:__KUBERNETES_SERVICE_PORT__",
CNI config: {
  "name": "linkerd-cni",
  "type": "linkerd-cni",
  "log_level": "info",
  "policy": {
      "type": "k8s",
      "k8s_api_root": "https://10.96.0.1:443",
      "k8s_auth_token": "__SERVICEACCOUNT_TOKEN__"
  },
  "kubernetes": {
      "kubeconfig": "/etc/cni/net.d/ZZZ-linkerd-cni-kubeconfig"
  },
  "linkerd": {
    "incoming-proxy-port": 4143,
    "outgoing-proxy-port": 4140,
    "proxy-uid": 2102,
    "ports-to-redirect": [],
    "inbound-ports-to-ignore": [4190,4191],
    "outbound-ports-to-ignore": [],
    "simulate": false
  }
}
Created CNI config /host/etc/cni/net.d/10-default.conflist
Done configuring CNI. Sleep=true
```

### Stats

```bash
$ bin/linkerd stat deploy --all-namespaces
NAMESPACE     NAME                 MESHED   SUCCESS      RPS   LATENCY_P50   LATENCY_P95   LATENCY_P99   TLS
docker        compose                 0/1         -        -             -             -             -     -
docker        compose-api             0/1         -        -             -             -             -     -
emojivoto     emoji                   1/1   100.00%   1.9rps           1ms          20ms          28ms    0%
emojivoto     vote-bot                1/1         -        -             -             -             -     -
emojivoto     voting                  1/1    87.50%   0.9rps           1ms           8ms          10ms    0%
emojivoto     web                     1/1    90.52%   1.9rps           8ms         100ms         180ms    0%
kube-system   kube-dns                0/1         -        -             -             -             -     -
linkerd       linkerd-controller      1/1         -        -             -             -             -     -
linkerd       linkerd-grafana         1/1   100.00%   0.3rps           1ms           3ms           3ms    0%
linkerd       linkerd-prometheus      1/1   100.00%   2.0rps           8ms          41ms          48ms    0%
linkerd       linkerd-web             1/1         -        -             -             -             -     -
```